packages/api/prisma/migrations/20230624131327_add_language_users/migration.sql

```diff
@@ -0,0 +1,17 @@
+-- CreateEnum
```

Comment:
Please remind me of what the command is to migrate the DB? `db.md` does not clearly state the command, and the several different formats I tried all caused errors.

Reply:
Run `nx prisma api migrate dev`. I'll update the db docs

Reply:
I was missing the `dev` part! I'll try again when I'm home

Reply:
Are you updating the docs in this PR or somewhere else?